server/src/postgis/aircraft.rs

```diff
@@ -295,16 +296,17 @@ pub async fn update_aircraft_position(aircraft: Vec<AircraftPosition>) -> Result
         PostgisError::Aircraft(AircraftError::DBError)
     })?;
 
+    let table_name = format!("{}.aircraft", super::PSQL_SCHEMA);
     let stmt = transaction
-        .prepare_cached(
+        .prepare_cached(&format!(
             "
-        INSERT INTO arrow.aircraft (identifier, geom, last_position_update)
+        INSERT INTO {table_name} (identifier, geom, last_position_update)
```

Comment:
When using templated queries, it's saver to add the double quotes around the table and column names, making sure you're not invoking a function or using a reserved word.
You can use the r#"" notation to add double quotes without escaping them to keep it readable.
Example can be found in the [`svc-storage` code](https://github.com/Arrow-air/svc-storage/blob/133e4bbe57ba2b852797a74d6e4d3b8cc94a7539/server/src/postgres/queries.rs#L230)

Reply:
https://github.com/Arrow-air/svc-gis/pull/18/commits/0846dfb6de39162bef367ca775ff602305a8b0fd